client/blocks/login/continue-as-user.jsx

```diff
@@ -50,23 +51,37 @@ function ContinueAsUser( { currentUser, redirectUrlFromQuery } ) {
 
 	const userName = currentUser.display_name || currentUser.username;
 
+	const handleChangeAccount = e => {
+		e.preventDefault();
+		onChangeAccount();
+	};
+
 	// Render ContinueAsUser straight away, even before validation.
 	// This helps avoid jarring layout shifts. It's not ideal that the link URL changes transparently
 	// like that, but it is better than the alternative, and in practice it should happen quicker than
 	// the user can notice.
-	const redirectLink = (
-		<a href={ validatedRedirectUrl || '/' }>
-			<Gravatar user={ currentUser } size={ 16 } />
-			{ userName }
-		</a>
-	);
 
 	return (
 		<div className="continue-as-user">
-			{ translate( 'or continue as {{userName/}}', {
-				components: { userName: redirectLink },
-				comment: 'Alternative link under login header, skips login to continue as current user.',
-			} ) }
+			<Gravatar user={ currentUser } imgSize={ 200 } size={ 200 } />
+			<Button primary href={ validatedRedirectUrl || '/' }>
+				{ translate( 'Continue as %(userName)s', { args: { userName } } ) }
+			</Button>
+			<p>
+				{ translate( 'Not %(userName)s? Log in with {{link}}another account{{/link}}', {
+					components: {
+						link: (
+							<a
+								href="#/"
```

Comment:
Does [`Button`](https://wpcalypso.wordpress.com/devdocs/design/button) component have any state that would look like a link but not need `href`?

Reply:
I guess this should just be a `<button />` if there's no need for a `href`.

Reply:
Yes, that's true. Using `<a/>` here is already not good for a11y because clicking it doesn't change the URL. There is some `is-link` class for `@automattic/components` `<Button/>` but it seems to be [deprecated](https://github.com/Automattic/wp-calypso/blob/master/packages/components/src/button/style.scss#L298) long time ago and it's  also causing alignment issues if used.

Reply:
Fixed in 61539507267ce33c43d23e525a06dd3626a23985